### PR TITLE
Add possibility to use postman variables in baseurl and path

### PR DESCRIPTION
**Pull request by devtribe**
Allow usage of {{var}} to use https://learning.getpostman.com/docs/postman/environments_and_globals/variables/

addresses #98

### CODE REVIEW

File: restdocs-api-spec-postman-generator/src/main/kotlin/com/epages/restdocs/apispec/postman/PostmanCollectionGenerator.kt [hunk 1]
```diff
@@ -73,41 +73,53 @@ object PostmanCollectionGenerator {
...
+                true -> null
+                else -> baseUrl.protocol
+            }
+            host = baseUrl.host
+            port = when (baseUrl.port) {
+                -1 -> null
+                else -> baseUrl.port.toString()
             }
-            path = baseUri.path + modelsWithSamePathAndMethod.first().request.path.replace(Regex("\\{[^/]+}")) {
+            path = baseUrl.path + modelsWithSamePathAndMethod.first().request.path.replace(Regex("(?<!\\{)\\{([^}]+)\\}(?!\\})")) {
```
Comment (mduesterhoeft) [inline restdocs-api-spec-postman-generator/src/main/kotlin/com/epages/restdocs/apispec/postman/PostmanCollectionGenerator.kt]: Can you explain why the change in the regex is needed? Can you also please add a comment explaining what it does.

Comment (devtribe) [reply restdocs-api-spec-postman-generator/src/main/kotlin/com/epages/restdocs/apispec/postman/PostmanCollectionGenerator.kt]: With the current regex, if we allow the postman variables `{{foo}}` in the path, it will replaced to `{:foo}`. So the new regex identify the difference between one and two curly brackets and only replace one curly bracket to colon.
E.g. `{foo}` will be replaced to `:foo` and `{{bar}}` stays the same.

See the test `should allow postman variable as part of the path` in https://github.com/ePages-de/restdocs-api-spec/pull/99/files/45969c105da0219eabff6cda2ae53907353a3a8b#diff-5ddef866bfe4db14f10c446ba410269dR123

Comment (mduesterhoeft) [reply restdocs-api-spec-postman-generator/src/main/kotlin/com/epages/restdocs/apispec/postman/PostmanCollectionGenerator.kt]: Sounds very reasonable. Thanks.
